### PR TITLE
[BPK-1457] Replace bindings with arrow syntax

### PR DESCRIPTION
**Pull request by georgegillams**

### CODE REVIEW

File: packages/bpk-component-image/src/withLazyLoading.js [hunk 1]
```diff
@@ -52,11 +52,6 @@ export default function withLazyLoading(
     constructor(): void {
       super();
 
-      this.setInView = this.setInView.bind(this);
-      this.removeEventListeners = this.removeEventListeners.bind(this);
-      this.checkInView = this.checkInView.bind(this);
-      this.isInViewPort = this.isInViewPort.bind(this);
-      this.supportsPassiveEvents = this.supportsPassiveEvents.bind(this);
```
[the remark this answers is not in the document]
Comment (georgegillams) [reply packages/bpk-component-image/src/withLazyLoading.js]: Just a mistake on my part :)

[the remark this answers is not in the document]
Comment (k0nserv) [reply packages/bpk-component-image/src/withLazyLoading.js]: So now too many methods have changed here. Namely

+ `componentWillUnmount`
+  `getPassiveArgs`

File: packages/bpk-react-utils/src/Portal.js [hunk 2]
```diff
@@ -38,14 +38,6 @@ class Portal extends Component {
     // shouldClose is used to keep track of the user's mouse-down events in order to
     // prevent the dialog closing if the mouse leaves / enters the portal during the click
     this.shouldClose = false;
-
-    this.close = this.close.bind(this);
```
Comment (k0nserv) [inline packages/bpk-react-utils/src/Portal.js]: Close has not been updated

File: packages/bpk-react-utils/src/Portal.js [hunk 3]
```diff
@@ -143,9 +135,9 @@ class Portal extends Component {
       isTargetClick,
       isPortalClick,
     };
-  }
+  };
 
-  getTargetElement() {
+  getTargetElement = () => {
```
Comment (k0nserv) [inline packages/bpk-react-utils/src/Portal.js]: This is not required

File: packages/bpk-react-utils/src/Portal.js [hunk 4]
```diff
@@ -155,15 +147,15 @@ class Portal extends Component {
     // implementation (see https://reactjs.org/docs/portals.html).
     // eslint-disable-next-line react/no-find-dom-node
     return this.props.target && findDOMNode(this);
-  }
+  };
 
-  getRenderTarget() {
+  getRenderTarget = () => {
```
Comment (k0nserv) [inline packages/bpk-react-utils/src/Portal.js]: This is also not required

File: packages/bpk-component-image/src/withLazyLoading.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (k0nserv) [reply packages/bpk-component-image/src/withLazyLoading.js]: This still isn't addressed @matteo-hertel

**Pull request merged**